flutter_inappwebview_android/android/src/main/java/com/pichillilorenzo/flutter_inappwebview_android/webview/in_app_webview/InAppWebViewClientCompat.java

```diff
@@ -309,12 +309,6 @@ public void onReceivedError(@NonNull WebView view,
   public void onReceivedError(WebView view, int errorCode, String description, String failingUrl) {
     final InAppWebView webView = (InAppWebView) view;
 
-    if (!WebViewFeature.isFeatureSupported(WebViewFeature.SUPPRESS_ERROR_PAGE) &&
```

Comment:
You can just remove !WebViewFeature.isFeatureSupported(WebViewFeature.SUPPRESS_ERROR_PAGE)  and keep the if